### ISSUE TITLE
About these two datasets: Avunue and Shanghai Tech

### ISSUE THREAD

**Issue by Smallwhite999** [donggong1/memae-anomaly-detection#20]
About these two datasets: Avunue and Shanghai Tech，can we train them directly or do we need to preprocess them first? If it is preprocessing, is the operation the same as the UCSD dataset?

Thanks！

**Comment by donggong1** [comment]
Yes. Training is straightforward on other datasets. Note that training on Shanghai Tech dataset takes more time. I will try to find time to clean the code for preparing the data files for other datasets.

**Comment by TanMing11** [comment]
> About these two datasets: Avunue and Shanghai Tech，can we train them directly or do we need to preprocess them first? If it is preprocessing, is the operation the same as the UCSD dataset?
> 
> Thanks！

Hello,
could you please send me the premodels about avenu and shanghaitech.
 I really need them. and my email is:  coldashes1@gmial.com or tanmingp11@163.com
Thanks!!!

**Comment by youngupup** [comment]
> and



> > About these two datasets: Avunue and Shanghai Tech，can we train them directly or do we need to preprocess them first? If it is preprocessing, is the operation the same as the UCSD dataset?
> > Thanks！
> 
> Hello,
> could you please send me the premodels about avenu and shanghaitech.
> I really need them. and my email is: [coldashes1@gmial.com](mailto:coldashes1@gmial.com) or [tanmingp11@163.com](mailto:tanmingp11@163.com)
> Thanks!!!

hello，do have the premodels about shanghaitech？i do not know deal with shanghaitech dataset by using matlab_script.Thanks

**Comment by youngupup** [comment]
> > About these two datasets: Avunue and Shanghai Tech，can we train them directly or do we need to preprocess them first? If it is preprocessing, is the operation the same as the UCSD dataset?
> > Thanks！
> 
> Hello,
> could you please send me the premodels about avenu and shanghaitech.
> I really need them. and my email is: [coldashes1@gmial.com](mailto:coldashes1@gmial.com) or [tanmingp11@163.com](mailto:tanmingp11@163.com)
> Thanks!!!

hello，do have the premodels about shanghaitech？i do not know deal with shanghaitech dataset by using matlab_script.Thanks
